src/main/java/com/example/Employee.java

```diff
@@ -0,0 +1,63 @@
+package com.example;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class Employee {
+    private String employeeId;
+    private String name;
+    private String position;
+    private double salary;
+
+    public Employee(String employeeId, String name, String position, double salary) {
+        this.employeeId = employeeId;
+        this.name = name;
+        this.position = position;
+        this.salary = salary;
+    }
+
+    public String getEmployeeId() {
+        return employeeId;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getPosition() {
+        return position;
+    }
+
+    public double getSalary() {
+        return salary;
+    }
+
+    public void promoteEmployee(String newPosition, double newSalary) {
+        this.position = newPosition;
+        this.salary = newSalary;
+    }
+
+    public double calculateYearlySalary() {
+        return this.salary * 12;
+    }
+
+    public void saveEmployee() {
+        String databaseUrl = "jdbc:mysql://localhost:3306/mydatabase";
```

Comment:
You could move this out of the function as a `private static final String` (or preferably have it read from an environment variable 😅)